index.js

```diff
@@ -9,29 +9,26 @@ function ToObject(val) {
 	return Object(val);
 }
 
-function ownEnumerableKeys(obj) {
-	var keys = Object.getOwnPropertyNames(obj);
-
-	if (Object.getOwnPropertySymbols) {
-		keys = keys.concat(Object.getOwnPropertySymbols(obj));
-	}
-
-	return keys.filter(function (key) {
-		return propIsEnumerable.call(obj, key);
-	});
-}
-
 module.exports = Object.assign || function (target, source) {
 	var from;
 	var keys;
 	var to = ToObject(target);
+	var symbols;
 
 	for (var s = 1; s < arguments.length; s++) {
-		from = arguments[s];
-		keys = ownEnumerableKeys(Object(from));
+		from = Object(arguments[s]);
+
+		for (var key in from) {
+			if (hasOwnProperty.call(from, key)) {
+				to[key] = from[key];
+			}
+		}
 
-		for (var i = 0; i < keys.length; i++) {
-			to[keys[i]] = from[keys[i]];
+		if (Object.getOwnPropertySymbols) {
```

Comment:
I wonder if the Symbols thing should be after or before the properties. The other `object.assign` polyfill has it the other way: https://github.com/ljharb/object.assign/blob/5dfb0414d2828539008ddc26119e69601bc84cbf/implementation.js#L21